src/index.html

```diff
@@ -12,7 +12,40 @@
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__img"></div>
+      <div class="card__container">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <ul class="card__stars">
+            <li class="card__stars-star"></li>
+            <li class="card__stars-star"></li>
+            <li class="card__stars-star"></li>
+            <li class="card__stars-star"></li>
+            <li class="card__stars-star"></li>
+          </ul>
```

Comment:
You should reuse your logic from previous `stars` task
It should be a block `stars` with modifiers `stars--n`